androidApp/build.gradle.kts

```diff
@@ -144,4 +144,9 @@ dependencies {
     androidTestImplementation(libs.hilt.android.testing)
 
     debugApi(libs.androidx.compose.ui.tooling)
+
+    implementation("com.google.accompanist:accompanist-systemuicontroller:0.34.0")
```

Comment:
Accompanist system UI controller has been deprecated, we should migrate to EdgeToEdge Api